sdk/spring/scripts/compatibility_delete_version.py

```diff
@@ -43,8 +43,35 @@ def delete_dependency_version(file_path):
         lines = pom_file.readlines()
     with open(file_path, 'w', encoding = 'utf-8') as new_pom_file:
         for line in lines:
-            if '<!-- {x-version-update;org.springframework' not in line:
+            if ';external_dependency} -->' not in line:
                 new_pom_file.write(line)
+            elif '<!-- {x-version-update;com.github.tomakehurst:wiremock-jre8;external_dependency} -->' in line:
+                new_pom_file.write(line)
+            elif external_dependencies_managed(line):
+                # listed in external-dependencies.txt but not managed by spring
+                new_pom_file.write(line)
+
+
+def external_dependencies_managed(line):
```

Comment:
i know it is a temporary fix, but we should store the dependencies managed by spring in a set or map

Reply:
We should not read the file for every single dependencies